### PR TITLE
Buttons in the repair section fixed

### LINKED ISSUE #291: Fix the text of "Book appointment" buttons
The alignment of "Book appointment" buttons in the repair and services pages look like this
image

I want to fix this

### PR DESCRIPTION
**Pull request by PoulavBhowmick03**
## Summary
Fixed the size and text orientation of buttons in the repair section
## Type of change
_Please select the relevant option._
- [X] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

## Checklist
_Please tick the relevant options._
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Screenshots

image

### CODE REVIEW

**Review by PoulavBhowmick03 — COMMENTED** [review]
Adjusted the buttons as asked

**Comment by PoulavBhowmick03** [comment]
@piyushlakheda

File: .vscode/settings.json
Comment (Neilblaze) [inline .vscode/settings.json:1]: Could you remove this unnecessary file? I would recommend adding vsc config to .gitignore from next time.

File: repair/style.css [hunk 1]
```diff
@@ -111,6 +111,19 @@
 .home-repair-col1 button:hover:after {
     opacity: 1;
   }
+  .home-repair-col button{
```
Comment (Neilblaze) [inline repair/style.css:114]: Why use the same CSS classes twice? Can we not combine both under a single class (`.home-repair-col button`)?

**Review by Neilblaze — APPROVED** [review]
LGTM!

**Pull request merged**